drbd-formula.spec

```diff
@@ -60,9 +57,9 @@ cp -R examples %{buildroot}%{fdir}/states/%{fname}/%{ftemplates}/
 cp -R form.yml metadata.yml pillar.example README.md %{buildroot}%{fdir}/metadata/%{fname}
 
 %files
-%defattr(-,root,root,-)
 %if 0%{?sle_version} < 120300
-%doc README.md LICENSE
+%license LICENSE
```

Comment:
The reason why we added this `if` statement is that before SLE12SP3 the `license` tag doesn't exist.
Actually, as we don't build anymore for versions less than SLE12SP3, I would remove this. We would have:
```
%doc README.md
%license LICENSE

```

Reply:
I made a mistake. Follow the spec cleaner without check too much, my bad to make the regression.